runtime/libtock_layout.ld

```diff
@@ -66,17 +66,18 @@ SECTIONS {
         LONG(ADDR(.data));
         LONG(SIZEOF(.bss));
         LONG(ADDR(.bss));
+
+        *(.start)
```

Comment:
Combining `.rt_header` and `.start` definitely deserves an explanation.

IIRC, something changed in the Rust toolchain, and with newer toolchain versions `elf2tab` no longer recognized `.rt_header` as being part of the binary. Combining them together fixes that (it's not clear whether that is a hack or the correct solution, but I think it's a hack).

Reply:
Unfortunately I don't remember why I had to make this change. I am pretty sure it was necessary for things to work, but I forget the details, and am having trouble finding any written explanation...

Reply:
Is there anything that should change here then?

Reply:
Done.